src/commands/commit/withHook/index.js

```diff
@@ -26,4 +27,13 @@ export const registerHookInterruptionHandler = () => {
   })
 }
 
+export const cancelIfRebasing = (mode: 'hook' | 'client') =>
```

Comment:
I think we can make this function `async` to benefit from this and be able to use `await` keyword inside of the fn

Reply:
see https://github.com/carloscuesta/gitmoji-cli/pull/445#discussion_r477316501